certif/translations/en.json

```
@@ -207,7 +207,8 @@
             }
           },
           "errors": {
-            "download": "Une erreur s'est produite pendant le téléchargement"
+            "download": "Une erreur s'est produite pendant le téléchargement",
+            "incorrect-type-file": "e format du fichier est incorrect, seul le format .csv est accepté"
```

C'est en anglais, donc voué à disparaitre mais il y a une typo:
```suggestion
            "incorrect-type-file": "Le format du fichier est incorrect, seul le format .csv est accepté"
```